### DISCUSSION TITLE
Can notes in HTML subtables be treated as footnotes (rather than subtable notes) in PDF?

### DISCUSSION THREAD

**Discussion by IULibScholComm** [quarto-dev/quarto-cli#8658] [Q&A] [answered]
### Description

One of my two HTML subtables has data with notes that I want to be footnotes. These notes are footnotes in the HTML and EPUB versions, but in the PDF the data notes are subtable notes (i.e. placed adjacent to the subtable rather than at the bottom of the page, given a letter identifier (ex. "a") rather than a number (ex. "1"), etc.) rather than footnotes. Can the notes in the subtable be treated as footnotes in the PDF version just as in the in the HTML and EPUB versions?

Reproducible Example:
````qmd
---
title: "test"
format: pdf
---

Text that has a footnote.^[This is a footnote.]

:::: {#tbl-test layout-ncol="2"}

::: {#tbl-test1}

```{=html}
<table>
  <thead>
    <tr>
      <th>test</th>
      <th>test</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>test</td>
      <td>test</td>
    </tr>
  </tbody>
</table>
```

Test1
:::

::: {#tbl-test2}

```{=html}
<table>
  <thead>
    <tr>
      <th>test<span data-qmd="^[This should be a footnote.]"></span></th>
      <th>test</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>test</td>
      <td>test</td>
    </tr>
  </tbody>
</table>
```

Test2
:::

Test
::::

````

I'm using Quarto 1.4.59 on Windows 10 Enterprise.

Thank you again very much for your help!

**Comment by cscheid** [comment, marked as answer]
Tracking the feature request [here](https://github.com/quarto-dev/quarto-cli/issues/8660)
